Lib/fontTools/feaLib/ast.py

```diff
@@ -117,14 +109,17 @@ def asFea(g):
 
 
 class Element(object):
+    """A base class representing "something" in a feature file."""
 
     def __init__(self, location=None):
+        #: location of this element - tuple of ``(filename, line, column)``
         self.location = location
 
     def build(self, builder):
```

Comment:
What does this method do?

Reply:
the base implementation does nothing, but subclasses may override to do things with builder